pkg/microservice/aslan/core/workflow/service/workflow/plugin_repo.go

```diff
@@ -126,18 +154,25 @@ func loadPluginRepoInfos(baseDir string, isOffical bool, readDir readDir, readFi
 	return resp, nil
 }
 
-func ListPluginRepositories(log *zap.SugaredLogger) ([]*commonmodels.PluginRepo, error) {
-	resp, err := commonrepo.NewPluginRepoColl().List()
+func ListUnofficalPluginRepositories(log *zap.SugaredLogger) ([]*commonmodels.PluginRepo, error) {
+	resp := []*commonmodels.PluginRepo{}
+	repos, err := commonrepo.NewPluginRepoColl().List()
 	if err != nil {
 		log.Errorf("list Plugin repos error: %v", err)
 		return resp, e.ErrListPluginRepo.AddDesc(err.Error())
 	}
+	for _, repo := range repos {
+		if repo.IsOffical {
+			continue
+		}
```

Comment:
why don't we use `is_offical ` as query condition to filter the repos when listing instead of filter by code？

Reply:
because bool do not have empty value. i change it to reference anyway.